messages/en.json

```diff
@@ -7,6 +7,7 @@
   "admin/b2b-organizations.selected-rows": "Selected rows: {qty}",
   "admin/b2b-organizations.showRows": "Show rows",
   "admin/b2b-organizations.of": "of",
+  "admin/b2b-organizations.invalid-schema": "B2B user creation is currently disabled because the MasterData CL (user profile) schema is invalid on this account. Please see the B2B Suite documentation (Frequently Asked Questions) for more information.",
```

Comment:
Please make "B2B Suite documentation" a hyperlink that opens https://developers.vtex.com/vtex-developer-docs/docs/vtex-b2b-suite in a new tab.